modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -1050,32 +1050,6 @@ TEST_P(CropTest, AccuracyTest)
     }
 }
 
-TEST_P(CopyTest, AccuracyTest)
-{
-    cv::Size sz_out = sz;
-    if (dtype != -1)
-    {
-        out_mat_gapi = cv::Mat(sz_out, dtype);
-        out_mat_ocv = cv::Mat(sz_out, dtype);
-    }
-
-    // G-API code //////////////////////////////////////////////////////////////
-    cv::GMat in;
-    auto out = cv::gapi::copy(in);
-
-    cv::GComputation c(in, out);
-    c.apply(in_mat1, out_mat_gapi, getCompileArgs());
-    // OpenCV code /////////////////////////////////////////////////////////////
-    {
-        cv::Mat(in_mat1).copyTo(out_mat_ocv);
-    }
-    // Comparison //////////////////////////////////////////////////////////////
-    {
-        EXPECT_EQ(0, cvtest::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
-        EXPECT_EQ(out_mat_gapi.size(), sz_out);
-    }
-}
```

Comment:
I'd keep it. We still need to validate that this functionality works

Reply:
I moved this test to streaming_tests.cpp. To keep this code here we need copy kernel to stay in core.hpp

Reply:
Done